FAQ: block and object storage (#250)
LambdaLabs/lambda-docs: content/en/cloud/block-object-storage.md

---
title: "Does Lambda GPU Cloud offer block or object storage?"
type: docs

> aliases:
> - guides/use-persistent-storage

Reviewer: We don't need a [redirect](https://gohugo.io/content-management/urls/#aliases), so we don't need these lines.

```suggestion
```

---
Lambda GPU Cloud currently doesn't offer block or object storage.

> Persistent storage needs to be created [using the dashboard](https://docs.lambdalabs.com/cloud/cloud-dashboard/#create-and-manage-persistent-storage-file-systems) and attached when
> you [create an instance](https://docs.lambdalabs.com/cloud/cloud-dashboard/#launch-restart-and-terminate-instances).

Reviewer: You want to use the [relref shortcode](https://gohugo.io/content-management/cross-references/) when linking to internal (docs site) pages. I'll explain why next time we meet.

Also, in most cases for readability, it's best to keep lines no longer than ~78 characters in length.

```suggestion
[using the dashboard]({{< relref "cloud-dashboard/#create-and-manage-persistent-storage-file-systems" >}})                                                                                                                                                    
and attached when you                                                                                                                                                                                                                                         
[create an instance]({{< relref "cloud-dashboard/#launch-restart-and-terminate-instances" >}}).  
```

> 

Reviewer: This whitespace (empty line) isn't needed.

```suggestion
```
